### PR TITLE
Feature/sling 9057 update javax activation

### PR DESCRIPTION
**Pull request by ugocei**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -35,7 +35,7 @@
     <description>The Apache Sling javax.activation bundle contributes an OSGi-compatible activation bundle.</description>
     
     <properties>
-        <javax.activation.version>1.1.1</javax.activation.version>
+        <javax.activation.version>1.2.0</javax.activation.version>
```
Comment (kwin) [inline pom.xml]: 1.2.1 is the newest version with a better license (EDL)

File: pom.xml [hunk 2]
```diff
@@ -63,8 +63,8 @@
     </build>
     <dependencies>
         <dependency>
-            <groupId>javax.activation</groupId>
-            <artifactId>activation</artifactId>
+            <groupId>com.sun.activation</groupId>
```
Comment (kwin) [inline pom.xml:66]: the newest version is available from https://search.maven.org/artifact/com.sun.activation/jakarta.activation/1.2.1/jar.

**Review by kwin — CHANGES_REQUESTED** [review]

**Review by kwin — APPROVED** [review]
LGTM now. We must not forget to remove the additional notice and license file from https://github.com/apache/sling-org-apache-sling-javax-activation/tree/master/src/main/appended-resources/META-INF.

**Comment by bosschaert** [comment]
This is a fairly old PR at this stage, but I see that @kwin requested additional changes:

@kwin wrote:
> We must not forget to remove the additional notice and license file from https://github.com/apache/sling-org-apache-sling-javax-activation/tree/master/src/main/appended-resources/META-INF.

@ugocei would you have time to update the branch with these changes?

**Comment by kwin** [comment]
1.2.2 is available meanwhile...

**Comment by kwin** [comment]
I will pick this up next week.

**Pull request closed without merging**